src/main/java/net/snowflake/ingest/streaming/internal/ParquetValueParser.java

```diff
@@ -17,6 +17,17 @@
 
 /** Parses a user column value into Parquet internal representation for buffering. */
 class ParquetValueParser {
+
+  // Parquet uses BitPacking to encode boolean, hence 1 bit per value
+  public static final float BIT_ENCODING_BYTE_LEN = 1.0f / 8;
+
+  /* (on average) 2 bytes / 8 values for the RLE+bitpack encoded definition level.
+  Since we don't have nested types, repetition level is always 0 and is not stored at all by Parquet. */
```

Comment:
Could you add details about justification that we discussed? that we have 2 cases:
- repeat count + repeated value bits
- value count + bits of values
with links to parquet class and methods
add byte/bit examples